README.md

```diff
@@ -268,8 +267,8 @@ Cloud Build will deploy:
 Once the build process is complete, navigate to the
 [agones-deploy-pipeline](https://console.cloud.google.com/deploy/delivery-pipelines/us-central1/global-game-agones-gameservers)
 delivery pipeline to review the rollout status. Cloud Build will create a Cloud Deploy release which automatically
-deploys the game server Agones Fleet to the `asia-east1` region first.
-The Fleet can be deployed to the next region in the queue via pressing the
+deploys the game server to the entire Agones Fleet, starting with the `asia-east1` region first.
```

Comment:
```suggestion
deploys the game server Fleet to the global set of Agones Clusters sequentially using [Cloud Deploy Automations](https://cloud.google.com/deploy/docs/automation), starting with the `asia-east1` region first, and with a (by default) 300s wait in between each region.
```

Something like that to specific.

Then I also think we can drop the manual steps as well, since they aren't relevant anymore. WDYT?